test/circuit-breakers.test.js

```diff
@@ -40,34 +39,51 @@ describe('circuit breakers', function(){
 			breaker.numBuckets = 1;
 			breaker.volumeThreshold = 1;
 
+			return app.listen(4242);
 		});
 
-		it('should trip circuit if service fails', function(done){
-			fetch('http://localhost:3000/__fail')
+		afterEach(function() {
+			// HACK: Reset the breaker
+			breaker._buckets = [breaker._createBucket()];
```

Comment:
this should probably simulate the upstream API getting healthy again…… but i couldn't figure out how to make it recover…… probably because i'm tired but maybe a bug……